apt.sources/cjdns.list

```diff
@@ -0,0 +1,7 @@
+#
+# A cjdns custom build package repo provided by a third party
+#
+# This is only used by the cjdns package install hack
+#
+
+deb https://ban.ai/raspbian/ cjdns cjdns
```

Comment:
It is on my list to publish the packages I built... just don't know when I will get to it